flow/raster_cache.cc

```diff
@@ -47,8 +47,18 @@ void RasterCacheResult::draw(SkCanvas& canvas, const SkPaint* paint) const {
 #endif
   canvas.resetMatrix();
   flow_.Step();
+  canvas.save();
+  // Make sure raster cache doesn't bleed to physical pixels outside of original
+  // bounds. https://github.com/flutter/flutter/issues/110002
+  if (bounds.fLeft + image_->dimensions().width() >
```

Comment:
nit: pull this into a variable, gaurd the save/clip and restore with it

Reply:
Done.